frontend/src/components/borrow/MyLoans.vue

```diff
@@ -34,10 +48,10 @@ const Component = Vue.extend({
         description,
         parameters: {interestPermil},
         blockchainState: {loanState}
-      }) => ({
+      }): ViewLoan => ({
```

Comment:
why we are only adding types to elements of loans list?
I think you may wanted to add types to _loans

Reply:
As far as I understand this infers the type, but I'll add the signature to `_loans`, too.

Reply:
my mistake. You are right ofc